### ISSUE TITLE
[BUG] test_orc_column_name_with_dots fails in nightly EGX tests

### ISSUE THREAD

**Issue by jlowe** [NVIDIA/spark-rapids#8735]
test_orc_column_name_with_dots is failing in the nightly EGX tests due to a data mismatch.  For example:
```
=========================== short test summary info ============================
01:16:13  FAILED ../../src/main/python/orc_test.py::test_orc_column_name_with_dots[{'spark.rapids.sql.format.orc.reader.type': 'PERFILE'}][IGNORE_ORDER]
01:16:13  FAILED ../../src/main/python/orc_test.py::test_orc_column_name_with_dots[{'spark.rapids.sql.format.orc.reader.type': 'MULTITHREADED', 'spark.rapids.sql.reader.multithreaded.combine.sizeBytes': '0', 'spark.rapids.sql.reader.multithreaded.read.keepOrder': True}][INJECT_OOM, IGNORE_ORDER]
01:16:13  FAILED ../../src/main/python/orc_test.py::test_orc_column_name_with_dots[{'spark.rapids.sql.format.orc.reader.type': 'COALESCING'}][INJECT_OOM, IGNORE_ORDER]
01:16:13  FAILED ../../src/main/python/orc_test.py::test_orc_column_name_with_dots[{'spark.rapids.sql.format.orc.reader.type': 'MULTITHREADED', 'spark.rapids.sql.reader.multithreaded.combine.sizeBytes': '64m', 'spark.rapids.sql.reader.multithreaded.read.keepOrder': True}][IGNORE_ORDER]
01:16:13  FAILED ../../src/main/python/orc_test.py::test_orc_column_name_with_dots[{'spark.rapids.sql.format.orc.reader.type': 'MULTITHREADED', 'spark.rapids.sql.reader.multithreaded.combine.sizeBytes': '64m', 'spark.rapids.sql.reader.multithreaded.read.keepOrder': False}][IGNORE_ORDER]
01:16:13  [31m= [31m[1m5 failed[0m, [32m17883 passed[0m, [33m1357 skipped[0m, [33m659 xfailed[0m, [33m246 xpassed[0m, [33m69 warnings[0m[31m in 22063.43s (6:07:43)[0m[31m =[0m
```

Looking at the output data from one of the failures, the GPU is reading 8 rows as nulls that the CPU does not.

**Comment by jlowe** [comment]
Appears to be a libcudf issue, filed rapidsai/cudf#13709.

**Comment by jlowe** [comment]
Fixed by rapidsai/cudf#13709.
